src/MapChart.tsx

```diff
@@ -21,13 +21,25 @@ import {
 
 const MapChart = (props: MapChartProps): JSX.Element => {
   const { stateManager } = props;
-  const [state] = stateManager;
+  const [state, dispatch] = stateManager;
   const { combinedMarkers } = state;
 
   const [mousePosition, setMousePosition] = useState({
```

Comment:
Until I figure out how to replace ZoomableGroup's onMoveEnd with a new lambda that uses dispatch, I will have to keep the useState here since it seems to magically record the zoom level (despite the Position interface not having a zoom prop).

Reply:
The fix was to put the dispatch call inside a function and pass that function to the onMoveEnd property.